src/build/build.ts

```diff
@@ -17,15 +17,19 @@ import * as vfs from 'vinyl-fs';
 
 import {HtmlProject} from './html-project';
 import {StreamResolver} from './stream-resolver';
+import {optimizePipe} from './util';
+import * as path from 'path';
```

Comment:
style: I've taken to putting external imports above internal, separated by a newline